.jenkins/infrastructure/build_azure_managed_images.Jenkinsfile

```diff
@@ -7,7 +7,7 @@ import java.time.format.DateTimeFormatter
 OECI_LIB_VERSION = env.OECI_LIB_VERSION ?: "master"
 oe = library("OpenEnclaveCommon@${OECI_LIB_VERSION}").jenkins.common.Openenclave.new()
 
-GLOBAL_TIMEOUT_MINUTES = 240
+GLOBAL_TIMEOUT_MINUTES = 480
```

Comment:
Why is 480 a better number? Isn't 8 hours a really long time?

Reply:
Yup, it is a really long time. But we can cancel the job if need be, this is the Jenkins job that rebuilds all the infrastructure and it takes a long time. Rhel, for example, takes forever: https://oe-jenkins-tf.westeurope.cloudapp.azure.com/job/CI-CD_Infrastructure/job/OpenEnclave-Rebuild-Infrastructure/